### DISCUSSION TITLE
Capturing user inputs and storing them as a parameter

### DISCUSSION THREAD

**Discussion by muammaramer** [FlowiseAI/Flowise#907] [Ideas]
It would be great if it's possible to have a way to capture the user's input in the chat and perform some actions based on that. This will have a lot of applications, such as using the input later as a parameter in the workflow or in webhooks. 

Example scenario:
- Bot: Are you asking about a specific product? 
- User: Do you have the product XYZ?
- Bot: We don't have the product XYZ for now.

At this point, we can capture the user input and if there is no result, we can trigger some actions (e.g. notify the sales team through an external API).

**Comment by chungyau97** [comment]
Hi @muammaramer,

This will be an enhancement for Flowise to achieve what you're looking for.

The current workaround will be using Zapier/Pipedream with 2 Flowise's Flow to achieve it.
|Flow|Input/Output|
|----|----|
|First Flow Input|Do you have the product XYZ?|
|First Flow Output|We don't have the product XYZ for now.|

|Flow|Input/Output|
|----|----|
|Second Flow Input|First Flow Output|
|Second Flow Output|Trigger some actions|

**Comment by mdlmarkham** [comment]
Is this feature available now? I'd like for an agent to be able to help the user write a report, and when it's done commit it to a database.
